mlserver/settings.py

```diff
@@ -393,3 +399,6 @@ def version(self) -> Optional[str]:
     # However, it's also possible to override them manually.
     parameters: Optional[ModelParameters] = None
     """Extra parameters for each instance of this model."""
+
+    cache_enabled: Optional[bool] = False
```

Comment:
I think the logic here should be "whatever the global setting says, unless I override it at the model level". With that in mind, should we set the default to `None` (which would effectively mean "whatever `Settings.cache_enabled` says)?

We could also clarify the comment saying that this is meant to override the global setting and won't enable it if the global is disabled.

Reply:
Updated in [43dfc1c](https://github.com/SeldonIO/MLServer/pull/1440/commits/43dfc1c7587e738d377d437761fcb2bcade94854)